lib/scheduled-list.js

```diff
@@ -423,6 +424,15 @@
           evt.preventDefault();
           this._reorderEnd(evt);
           break;
+        case 'click':
+          if (!this.editing) {
+            var li = evt.target.closest('li');
+            var dialog = document.querySelector(li.dataset.for);
+            var text = li.querySelector('h3').textContent;
+            dialog.textContent = text + ' item clicked!';
+            dialog.open(evt);
```

Comment:
The scheduled-list lib should not know about the dialogs, we should maybe dispatch a `CustomEvent` instead.
Like we do for `top-reached` or `hidden-new-content`.

I'd suggest an `item-selected` event, dispatched on `this.list` with the `index` of the selected item in the `detail` field.
Then the dialog code can move into `app.js`.